<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usermng</class>
 <widget class="QWidget" name="usermng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="m_ctlFilter">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 85, 255);</string>
     </property>
     <property name="placeholderText">
      <string>按名字查询</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>80</number>
     </attribute>
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>登录用户</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>登录密码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户角色</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户信息</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>151</height>
      </size>
     </property>
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>登录用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>登录密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txPasswrd"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>用户角色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbUserRole">
          <item>
           <property name="text">
            <string>管理员</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>开发人员</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>测试人员</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>一般用户</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>联系电话</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txUserTel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>用户信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txUserInfo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="bt_add">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_remove">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_modify">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
